kuma/wiki/models.py

```diff
@@ -2012,3 +2012,28 @@ class DocumentSpamAttempt(SpamAttempt):
 
     def __unicode__(self):
         return u'%s (%s)' % (self.slug, self.title)
+
+
+class BCSignal(models.Model):
+    """Model to keep track of the BC signals."""
+    document = models.ForeignKey(
+        Document,
+        related_name='bc_signals',
+        null=True,
+        blank=True,
+        verbose_name=_('Document (optional)'),
+        on_delete=models.SET_NULL
+    )
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL
+    )
+    submitted_at = models.DateTimeField(auto_now_add=True)
+
+    def __unicode__(self):
+        return u'BC Signal: %s' % (self.document.slug)
```

Comment:
I think it works with `__str__` just as well. One less thing to avoid having to rewrite when we upgrade to Python 3. 
Also, can you format the string with `.format` instead because I see that as a much more common tool.